src/main/java/seedu/address/logic/parser/FilterCommandParser.java

```diff
@@ -25,6 +25,8 @@ public class FilterCommandParser implements Parser<FilterCommand> {
     public static final String INCORRECT_AGE = "Age should be an integer";
     public static final String INCORRECT_DATE_FORMAT = "Appointment date should be form dd/MM/yyyy";
     public static final String INCORRECT_RANGE = "Please provide range in format: Lower bound - Upper bound";
```

Comment:
Is it necessary to keep INCORRECT_RANGE?

Reply:
Yes it's used for appointment range also. I can remove INCORRECT_AGE tho